public/content/translations/cs/developers/docs/intro-to-ether/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Úvod ke kryptoměně ether
+description: Úvod ke kryptoměně ether pro vývojáře.
+lang: cs
+---
+
+## Předpoklady {#prerequisites}
+
+K lepšímu pochopení této stránky vám doporučujeme si nejprve přečíst náš [úvod do Etherea](/developers/docs/intro-to-ethereum/).
+
+## Co je kryptoměna? {#what-is-a-cryptocurrency}
+
+Kryptoměna je médium směny, které je zajištěno účetní knihou založenou na blockchainové technologii.
+
+Médium směny je cokoliv, co je široce přijímáno jako platba za zboží a služby, a účetní kniha je datové úložiště, které sleduje transakce. Blockchainová technologie umožňuje uživatelům provádět transakce v účetní knize bez nutnosti spoléhat se na důvěryhodnou třetí stranu, která by účetní knihu udržovala.
+
+První kryptoměnou byl Bitcoin, který vytvořil Satoshi Nakamoto. Od spuštění Bitcoinu v roce 2009 lidé vytvořili tisíce kryptoměn na různých blockchainech.
+
+## Co je Ether? {#what-is-ether}
+
+**Ether (ETH)** je kryptoměna používaná pro různé účely v síti Ethereum. V základu je to jediná přijatelná forma platby za transakční poplatky a po [Sloučení](/roadmap/merge) je ether vyžadován k validaci a navrhování bloků na hlavní síti. Ether se také používá jako primární forma zajištění na [DeFi](/defi) půjčkových trzích, jako účetní jednotka na trzích NFT, jako platba za poskytování služeb nebo prodej reálného zboží atd.
+
+Ethereum umožňuje vývojářům vytvářet [**decentralizované aplikace (dappky)**](/developers/docs/dapps), které sdílejí společný fond výpočetního výkonu. Tento sdílený fond je omezený, takže Ethereum potřebuje mechanismus, který určí, kdo ho může používat. Jinak by mohla dappka omylem nebo záměrně spotřebovat všechny síťové zdroje, což by znemožnilo přístup ostatním.
+
+Kryptoměna ether podporuje mechanismus stanovení cen pro výpočetní výkon Etherea. Když uživatelé chtějí provést transakci, musí zaplatit ethereum, aby byla jejich transakce na blockchainu uznána. Tyto náklady jsou známé jako [poplatky za palivo](/developers/docs/gas/) a výše poplatku závisí na množství výpočetního výkonu potřebného k provedení transakce a na celosíťové poptávce po výpočetním výkonu v daném okamžiku.
```

Comment:
Per @konopkja, "etherem" is an acceptable translation for "ether" (it is not a mis-spelling of "Ethereum" as the markdown checker suggests). Reverting these.

```suggestion
Kryptoměna ether podporuje mechanismus stanovení cen pro výpočetní výkon Etherea. Když uživatelé chtějí provést transakci, musí zaplatit etherem, aby byla jejich transakce na blockchainu uznána. Tyto náklady jsou známé jako [poplatky za palivo](/developers/docs/gas/) a výše poplatku závisí na množství výpočetního výkonu potřebného k provedení transakce a na celosíťové poptávce po výpočetním výkonu v daném okamžiku.
```